config/drupal-phpunit-bootstrap-file.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+/**
+ * @file
+ *
+ * This fixes Drupal testing namespace autoloading and PHPUnit compatibility.
+ */
+
+use DrupalFinder\DrupalFinder;
+use Rector\Core\Autoloading\BootstrapFilesIncluder;
+use Symplify\PackageBuilder\Parameter\ParameterProvider;
+
+assert($this instanceof BootstrapFilesIncluder);
+/** @phpstan-ignore-next-line */
+$parameterProvider = $this->parameterProvider;
+assert($parameterProvider instanceof ParameterProvider);
```

Comment:
This assertion failed. It's probably due to rector-prefixed. But our workflows _should_ be mimicking that.

Reply:
I verified the workflows are using rector-prefixed and the notice wasn't thrown. But, that's because the PHP environment isn't configured to catch assertions, I believe.

<img width="576" alt="Screen Shot 2021-04-16 at 8 08 16 AM" src="https://user-images.githubusercontent.com/3698644/115028807-e899c880-9e8a-11eb-8da1-aa780c5d2b98.png">

So instead of assert, we can use regular doc type hinting and an `if NULL`

Reply:
Fixed with https://github.com/palantirnet/drupal-rector/pull/134/commits/4f498c8d8741b2f1661fdc01b4b3a9c20b4baf8c

Reply:
Confirming Fixed